docs/myst.yml

```diff
@@ -21,7 +21,7 @@ project:
     - file: reference/index.md
       children:
       - pattern: 'reference/!(index)*.md'
-    - file: developer/index.md
+    - title: Developer documentation
```

Comment:
Unintended change?

Reply:
There is no `developer/index.md` so I saw an error! Do you want me to make another PR for this one too?

Reply:
https://github.com/jupyter-book/myst-theme/pull/712